src/data/index.ts

```diff
@@ -207,6 +219,18 @@ const actions = {
 			isLoading,
 		};
 	},
+	/**
+	 * Set is updating action creator.
+	 *
+	 * @param {boolean} isUpdating True if UI is updating data.
+	 * @returns {object} Set updating action object.
+	 */
+	setIsUpdating ( isUpdating: boolean ): SetIsUpdatingAction {
```

Comment:
Some inconsistent code style here, is that from an updated rule set?
```suggestion
	setIsUpdating( isUpdating: boolean ) : SetIsUpdatingAction {
```